search-engine/lib/condition/condition_is_boosterfun.rb

```diff
@@ -0,0 +1,39 @@
+class ConditionIsBoosterfun < Condition
+  def search(db)
+    # Limit booster fun treatments to known sets
+    valid_sets = ["eld", "thb", "iko", "m21", "znr", "khm", "stx", "afr", "mid", "vow", "neo", "snc", "dmu", "bro", "one", "2xm", "mh2", "clb", "cmr", "2x2", "dmr", "unf", "bot", "brr"]
```

Comment:
This isn't going to be fun to update every time a new set comes out

Reply:
Yeah I couldn't come up with a better way to do it. Maybe some combination of set type and release date, but I wasn't able to get it exactly working.